api/types.go

```diff
@@ -49,48 +50,6 @@ type PubsubScore struct {
 	Score *pubsub.PeerScoreSnapshot
 }
 
-type MinerInfo struct {
```

Comment:
Moved into the actor abstractions. We need a copy of the below constructor per-actor version.